QUEST_JOBSTEP.tsv

```diff
@@ -1638,260 +1638,260 @@ QUEST_JOBSTEP_20150317_001637	Krivis Master told you that Krivis siblings who we
-QUEST_JOBSTEP_20150317_001864	Collect well burnable stones
+QUEST_JOBSTEP_20150317_001862	The Fletcher Master told you that he would accept you as his disciple if you could bring him well burnable stones, but you can't really trust him. Defeat Yonazolems in the central district and collect well burnable stones.
+QUEST_JOBSTEP_20150317_001863	You've obtained well burnable stones. Hand them over to the Fletcher Master.
+QUEST_JOBSTEP_20150317_001864	Collect flammable stones
 QUEST_JOBSTEP_20150317_001865	Recruiting Challengers. [Schwarze Reiter]
-QUEST_JOBSTEP_20150317_001866	Call of Schwarzer Reiter Master
-QUEST_JOBSTEP_20150317_001867	Schwarze Reiter Master is looking for you at Woods of Attached Bridge
+QUEST_JOBSTEP_20150317_001866	Call of the Schwarzer Reiter Master
+QUEST_JOBSTEP_20150317_001867	The Schwarze Reiter Master is looking for you at Woods of Attached Bridge
 QUEST_JOBSTEP_20150317_001868	Duel against Schwarze Reiter Master
-QUEST_JOBSTEP_20150317_001869	Schwarze Reiter Master told you that both guns and swords are important, but you also need to possess hatred. Duel against Schwarze Reiter Master and win.
-QUEST_JOBSTEP_20150317_001870	Talk to Schwarze Reiter Master
-QUEST_JOBSTEP_20150317_001871	You've won against Schwarze Reiter Master. Talk with Schwarze Reiter Master.
+QUEST_JOBSTEP_20150317_001869	The Schwarze Reiter Master told you that the guns and the swords are important, but you also need to possess the hatred. Duel against Schwarze Reiter Master and win.
+QUEST_JOBSTEP_20150317_001870	Talk to the Schwarze Reiter Master
+QUEST_JOBSTEP_20150317_001871	You've won against the Schwarze Reiter Master. Talk with the Schwarze Reiter Master.
 QUEST_JOBSTEP_20150317_001872	Acknowlegement of Schwarze Reiter 
-QUEST_JOBSTEP_20150317_001873	Go to Cleric Master
-QUEST_JOBSTEP_20150317_001874	Go and talk to Cleric Master in Klaipeda
-QUEST_JOBSTEP_20150317_001875	Obtain Wildflower Stamen from Lonesome Marketplace
-QUEST_JOBSTEP_20150317_001876	Cleric Master told you that the herbs are running out so he wants Wildflower Stamen instead. Obtain Wildflower Stamen from Lonesome Marketplace and take it to Cleric Master.
-QUEST_JOBSTEP_20150317_001877	Return to Cleric Master
-QUEST_JOBSTEP_20150317_001878	Gathered Wildflower Stamen. Gave it to Cleric Master.
-QUEST_JOBSTEP_20150317_001879	Wildflower Stamen
+QUEST_JOBSTEP_20150317_001873	Go to Master Cleric
+QUEST_JOBSTEP_20150317_001874	Go and talk to Master Cleric in Klaipeda
+QUEST_JOBSTEP_20150317_001875	Obtain Wildflower Alcohol from Lonesome Marketplace
+QUEST_JOBSTEP_20150317_001876	Cleric Master told you that the herbs are running out so he wants wildflower alcohol instead. Obtain wildflower alcohol from Lonesome Marketplace and take it to Cleric Master.
+QUEST_JOBSTEP_20150317_001877	Return to Master Cleric
+QUEST_JOBSTEP_20150317_001878	Gathered wild flower staman. Give it to Cleric Master.
```

Comment:
Why is it uncapitalized? 
Also: Give it to Cleric Master -> Give it to the Cleric Master.